test/acceptance/lettuce/integration/terrain.py

```diff
@@ -50,7 +50,7 @@ def before_all():
     world.cb_config_to_start = ''
     world.cb_pid = get_cb_pid()
     world.bin_parms = None
-    drop_all_test_databases(world.config['mongo']['host'], int(world.config['mongo']['port']))
+    #drop_all_test_databases(world.config['mongo']['host'], int(world.config['mongo']['port']))
```

Comment:
Maybe a "Fixme: these steps are comment by {some reason}" could be added above this line?